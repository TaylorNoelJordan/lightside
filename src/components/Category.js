import React from 'react';
import './Category.scss';

const Category = ({title, changeView}) => {
    return (
        <div>
            <button className='selectCategoryBtn' onClick={changeView}>
                {title}

            </button>
        </div>
    )
}

export default Category;