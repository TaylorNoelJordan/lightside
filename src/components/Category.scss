.selectCategoryBtn {
    border-radius: 50%;
    color: black;
    height: 300px;
    width: 300px;
}

