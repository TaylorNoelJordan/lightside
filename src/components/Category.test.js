import React from 'react';
import Category from './components/Category';
import { shallow } from 'enzyme';

describe('Category', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Category />)
    });

    it('should render a button', () => {
        expect(wrapper).instanceOf().toMatchSnapShot();
    })
})