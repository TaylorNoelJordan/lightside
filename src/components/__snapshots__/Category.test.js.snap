// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category should render a button 1`] = `
<button
  className="selectCategoryBtn"
  onClick={[MockFunction]}
>
  <img
    alt=""
    src=""
  />
</button>
`;
